providers/org/datacite/normalizer.py

```diff
@@ -0,0 +1,585 @@
+import re
+import logging
+
+from share.normalize import ctx, tools
+from share.normalize.parsers import Parser
+from share.normalize.utils import format_doi_as_url
+
+
+logger = logging.getLogger(__name__)
+
+THE_REGEX = re.compile(r'(^the\s|\sthe\s)')
+DATE_REGEX = re.compile(r'(^[^a-zA-Z]+$)')
+
+
+def force_text(data):
```

Comment:
I don't why `None` is in the metadata. I haven't seen `None` in any required field for the `oai_datacite` prefix though (title is one of the required fields).